src/one_dragon/base/config/one_dragon_config.py

```
@@ -168,6 +169,33 @@ def current_instance_force_login(self) -> bool:
         instance = self.current_active_instance
         return instance is not None and instance.force_login_before_run
 
+    @property
+    def current_instance_should_force_login(self) -> bool:
+        """
+        判断当前激活的实例在一条龙运行前是否需要强制重新登录。
+
+        国服 / B服 / 国际服 是三个不同的游戏客户端，各自保留一套登录状态，
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)